doc/api.md

```diff
@@ -0,0 +1,120 @@
+# API
+
+## cartodb.DI.createDashboard(domObject, vizJson, callback(err, obj))
+Creates a new dashboard inside `domObject` based on `vizJson`.
+
+#### Arguments
+- domObject: dom object identifier where the dashboard will be inserted
+- vizJson: either a vizjson url or a vizjson object. See viz.json spec to see how to generate one
+- callback: it will be called when the dashboard is generated. First argument will be null if
+  everything was fine, an `Error` object otherwise. Second argument is a `Dashboard` object, see
+  below
+
+#### Example
+```js
+cartodb.deepInsights.createDashboard('#dashboard', vizJSONurl, function(err, dashboard) {
```

Comment:
We're missing [the options](https://github.com/CartoDB/deep-insights.js/pull/296/files#diff-e1d62d8c4965edf0e532dfa19da0891fR87) argument here.